### PR TITLE
Message delete event missing some author information

### PR DESCRIPTION
**Pull request by danny-may**
Fixes [Message delete logs often times do not properly get the author's username and discrim](https://discord.com/channels/194232473931087872/989345216442007582) and an issue I spotted where `b!log track <sender>` didnt work

### CODE REVIEW

File: src/database/stores/RethinkDbGuildStore.ts [hunk 1]
```diff
@@ -725,12 +725,13 @@ export class RethinkDbGuildStore implements GuildStore {
...
 
-    public async setLogIgnores(guildId: string, userIds: string[]): Promise<boolean> {
+    public async setLogIgnores(guildId: string, userIds: string[], ignore: boolean): Promise<boolean> {
         const guild = await this.#table.get(guildId);
         if (guild === undefined)
             return false;
 
-        if (!await this.#table.update(guildId, r => ({ logIgnore: r('logIgnore').default([]).setUnion(userIds) })))
+        const method = ignore ? 'setUnion' : 'setDifference';
+        if (!await this.#table.update(guildId, r => ({ logIgnore: r('logIgnore').default([])[method](userIds) })))
```
Comment (HunteRoi) [inline src/database/stores/RethinkDbGuildStore.ts:734]: any possibility this line can be made easier to understand? 😳

File: src/database/stores/RethinkDbGuildStore.ts [hunk 2]
```diff
@@ -725,12 +725,13 @@ export class RethinkDbGuildStore implements GuildStore {
         return true;
     }
 
-    public async setLogIgnores(guildId: string, userIds: string[]): Promise<boolean> {
+    public async setLogIgnores(guildId: string, userIds: string[], ignore: boolean): Promise<boolean> {
```
Comment (HunteRoi) [inline src/database/stores/RethinkDbGuildStore.ts:728]: you updated the signature of this method but I don't see any change in the `GuildStore` interface nor in the `log` command, do I? Why is that?

Comment (danny-may) [reply src/database/stores/RethinkDbGuildStore.ts:728]: The interface always had the `ignore:boolean` param, the impl just wasnt using it. whoops!

File: src/database/stores/RethinkDbGuildStore.ts (hunk 1 above)
Comment (danny-may) [reply src/database/stores/RethinkDbGuildStore.ts:734]: I could split it onto multiple lines? But its rethink, its not the most elegant of syntaxes in general

Comment (danny-may) [reply src/database/stores/RethinkDbGuildStore.ts:739]: Added comments

**Review by HunteRoi — APPROVED** [review]

**Pull request merged**